.goreleaser.yml

```diff
@@ -165,6 +174,15 @@ dockers:
       - "rustinliu/tichi-blunderbuss-plugin:{{ .Tag }}"
       - "rustinliu/tichi-blunderbuss-plugin:{{ .Major }}"
     dockerfile: ./deployments/plugins/blunderbuss/Dockerfile
+  - binaries:
```

Comment:
Please change the line, and please be consistent with the rest of the configuration.